pkg/gen/input/makefile/internal/file/makefile.go

```diff
@@ -75,7 +75,7 @@ package-linux: $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-linux-amd64.tar.gz
 $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-%-amd64.tar.gz: DIR=$(PACKAGE_DIR)/$<
 $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-%-amd64.tar.gz: $(APPLICATION)-v$(VERSION)-%-amd64
 	mkdir -p $(DIR)
-	cp $< $(DIR)
+	cp $< $(DIR)/$(APPLICATION)
```

Comment:
Do we need `$(APPLICATION)-v$(VERSION)-%-amd64` target at all then? I feels like this one can depend on `$(APPLICATION)-%` now